media/css/base/mozilla-share-cta.less

```diff
@@ -6,9 +6,7 @@
 
 @font-face {
     font-family: 'Font Awesome';
-    src: url('/media/fonts/fontawesome-webfont.eot?v4.01#iefix') format('embedded-opentype'),
-         url('/media/fonts/fontawesome-webfont.woff?v4.01') format('woff'),
-         url('/media/fonts/fontawesome-webfont.ttf?v4.01') format('truetype');
+    src: url('/media/fonts/fontawesome-webfont.woff?v4.01') format('woff');
```

Comment:
Good call. There are some other pages using the full Font Awesome, I'll update those as well.

Of course, in many of these cases it might be preferable to use a custom font with only the glyphs we need. Like for this sharing widget we only need four icons but we're loading the entire 84k font (woff2 will be a little smaller). We could also switch to SVG instead of icon fonts but that's a bigger can of worms.